### PR TITLE
AAG-16 #style improvements

### PR DESCRIPTION
**Pull request by anaumchik**
**Refactoring:**
- style changes

### CODE REVIEW

File: mvvm_example/src/main/java/com/distillery/android/blueprints/mvvm/todo/fragments/ToDoListFragment.kt [hunk 1]
```diff
@@ -89,7 +89,7 @@ class ToDoListFragment : Fragment() {
      * sets the divider line visibility by checking completed and non completed list items
      * if the only of the both list is empty then it hides the line otherwise makes it visible
      */
-    private fun setDividerVisibility() { // todo
+    private fun setDividerVisibility() { // TODO is this todo actual one?
```
Comment (anaumchik) [inline mvvm_example/src/main/java/com/distillery/android/blueprints/mvvm/todo/fragments/ToDoListFragment.kt:92]: is this actual todo or we can remove it?

Comment (dineshnukemars) [reply mvvm_example/src/main/java/com/distillery/android/blueprints/mvvm/todo/fragments/ToDoListFragment.kt:92]: This checks both list sizes and sets the divider visibility if one of the list is empty. In other words it only sets the divider visibility if both lists are not empty

Comment (dineshnukemars) [reply mvvm_example/src/main/java/com/distillery/android/blueprints/mvvm/todo/fragments/ToDoListFragment.kt:92]: Without this code u cannot make this divider line hide or show. But just now I realized this should be on the viewmodel not in the view

Comment (dineshnukemars) [reply mvvm_example/src/main/java/com/distillery/android/blueprints/mvvm/todo/fragments/ToDoListFragment.kt:92]: sry, I misunderstood the question. yes, we can remove it.

**Pull request merged**